app/src/main/java/com/example/dine_and_donate/ProfileActivity.java

```diff
@@ -89,17 +103,34 @@ public boolean onNavigationItemSelected(@NonNull MenuItem menuItem) {
         //Change bottom navigation profile icon to filled
         bottomNavigationView.getMenu().findItem(R.id.action_profile).setIcon(R.drawable.instagram_user_filled_24);
         setUpBottomNav();
+
+        //retrieve values from database
+        mRef.addValueEventListener(new ValueEventListener() {
+            @Override
+            public void onDataChange(DataSnapshot dataSnapshot) {
+                mIsOrg = (Boolean) dataSnapshot.child("isOrg").getValue();
+                setUpTopProfile(dataSnapshot.child("name").getValue().toString());
+            }
+            @Override
+            public void onCancelled(DatabaseError databaseError) {
+
+            }
+        });
     }
 
-    //set up for top of profile page
-    private void setUpTopProfile() {
-        //TODO change boolean to isOrg boolean user attribute to display correct profile
-        if(false) {
-            mLayoutForOrg.setVisibility(View.INVISIBLE);
-            mLayoutForConsumer.setVisibility(View.VISIBLE);
-        } else {
+    //set up for top of profile page based on user type
+    private void setUpTopProfile(String name) {
+       //display orgView when user type is an organization
+        if(mIsOrg) {
             mLayoutForOrg.setVisibility(View.VISIBLE);
             mLayoutForConsumer.setVisibility(View.INVISIBLE);
+            Log.d("name", name);
+            mOrgName.setText(name);
+        } else {
+            mLayoutForOrg.setVisibility(View.INVISIBLE);
```

Comment:
You probably want to use View.GONE here. View.INVISIBLE makes the layout still take up space http://tips.androidgig.com/invisible-vs-gone-view-in-android/